apps/wallet/src/ui/app/staking/stake/StakeForm.tsx

```diff
@@ -134,8 +135,7 @@ export function StakeFormComponent({ validatorAddress, epoch, onSuccess }: Stake
 
     const handleSubmit = async (_: FormValues, formikHelpers: FormikHelpers<FormValues>) => {
         try {
-            const response = await stakeTokenMutateAsync(formikHelpers);
-            onSuccess(response);
+            await stakeTokenMutateAsync(formikHelpers);
```

Comment:
you can also pass a onSuccess here, so it's not necessary to pass the formik helpers all the way down, just do this:

```suggestion
            await stakeTokenMutateAsync(undefined, {
                onSuccess: () => {
                    formikHelpers.resetForm()
                }
            });
```